### PR TITLE
npm: add missing typings and remove unused modules

### PR DESCRIPTION
**Pull request by sthaha**

### CODE REVIEW

File: ee_tests/package.json [hunk 1]
```diff
@@ -46,10 +47,9 @@
     "protractor-jasmine2-screenshot-reporter": "0.3.5",
     "rimraf": "^2.6.2",
     "ts-helpers": "1.1.2",
-    "ts-node": "3.0.2",
-    "tslint": "5.0.0",
+    "tslint": "5.8.0",
     "tslint-loader": "^3.5.0",
-    "typescript": "^2.3.0"
+    "typescript": "^2.6.0"
```
Comment (joshuawilson) [inline ee_tests/package.json]: we are currently using 2.5.3 every where else

Comment (sthaha) [reply ee_tests/package.json]: @joshuawilson thanks for the alert. I will submit a fix soon.

Comment (sthaha) [reply ee_tests/package.json]: @joshuawilson and should the version of node.js be 8.3.x ?

Comment (joshuawilson) [reply ee_tests/package.json]: yes, we are on Node v8.3.0 and npm v5.3.0

Comment (sthaha) [reply ee_tests/package.json]: @joshuawilson PR https://github.com/fabric8io/fabric8-test/pull/328 should fix this. Could you please review?

**Pull request merged**